Sources/CodeEditSourceEditor/Gutter/GutterView.swift

```diff
@@ -204,8 +242,9 @@ public class GutterView: NSView {
         CATransaction.begin()
         superview?.clipsToBounds = false
         superview?.layer?.masksToBounds = false
-        layer?.backgroundColor = backgroundColor?.cgColor
+//        layer?.backgroundColor = backgroundColor?.cgColor
```

Comment:
Is this left over or is it being kept for the future?

Reply:
No that was supposed to be removed, fixed